src/views/preview/project-view-hoc.jsx

```diff
@@ -0,0 +1,68 @@
+const bindAll = require('lodash.bindall');
+const React = require('react');
+const PropTypes = require('prop-types');
+const connect = require('react-redux').connect;
+
+const previewActions = require('../../redux/preview.js');
+
+/**
+ * Higher-order component for presenting a project view.
+ * @param  {React.Component} Component a project view component
+ * @return {React.Component}           a wrapped project view component
+ */
+
+const ProjectViewHOC = Component => {
+    // initialize Sentry instance, making sure it hasn't been initialized already
+    if (!window.Sentry && `${process.env.SENTRY_DSN}` !== '') {
+        const Sentry = require('@sentry/browser');
+        Sentry.init({
+            dsn: `${process.env.SENTRY_DSN}`,
+            // Do not collect global onerror, only collect specifically from React error boundaries.
+            // TryCatch plugin also includes errors from setTimeouts (i.e. the VM)
+            integrations: integrations => integrations.filter(i =>
+                !(i.name === 'GlobalHandlers' || i.name === 'TryCatch'))
+        });
+        window.Sentry = Sentry; // Allow GUI access to Sentry via window
+    }
+
+    class WrappedComponent extends React.Component {
+        constructor (props) {
+            super(props);
+            bindAll(this, [
```

Comment:
This does nothing anymore, so can be removed. Which means you can get rid of the wrapping component altogether, and just hook it up with redux. `connect(...)(Component)`.

Reply:
Done